cfgov/alerts/post_sqs_messages.py

```diff
@@ -55,26 +55,32 @@
 )
 
 
-def matching_issue(title, issues):
-    return next((issue for issue in issues if issue.title == title), None)
+def post_message(body, title, github_creds={}, mattermost_creds={}):
+    issue = GithubAlert(github_creds).post(
+        title=title,
+        body=body,
+    )
 
+    if mattermost_creds:
+        MattermostAlert(mattermost_creds).post(
+            text='Alert: {}. Github issue at {}'.format(
+                body,
+                issue.html_url,
+            )
+        )
 
-def post_to_chat(endpoint, username, message, issue_url):
-    text = 'Alert: {}. Github issue at {}'.format(
-        message,
-        issue_url,
-    )
-    payload = {
-        'text': text,
-        'username': args.mattermost_username,
-    }
-    requests.post(
-        args.mattermost_webhook_url,
-        data=json.dumps(payload),
+
+def cleanup_jenkins_message(message):
+    return message.replace(
+        '#', '# '  # Avoids erroneous Github issue link
```

Comment:
This seems like it could be potentially over greedy - is it possible that error messages might contain this character elsewhere? You could do something a bit more specific like:

```py
>>> import re
>>> msg = 'Error #123 where string contains # character'
>>> re.sub(r'#(\d+)', r'# \1', msg)
'Error # 123 where string contains # character'
```